### DISCUSSION TITLE
When to use action list vs. option list vs. default select? (feedback on `polaris.shopify.com/components/selection-and-input/select`)

### DISCUSSION THREAD

**Discussion by raquelbreternitz** [Shopify/polaris-react#11255] [Pattern documentation]
Is there thought / guidance on when to use an action list vs. option list vs. default selects? A mini-pattern or expanded documentation would be helpful here! TIA

**Comment by heyjoethomas** [comment]
Hey Raquel 👋 

1. `ActionList` is used for a menu of buttons/actions a merchant can take
2. `OptionList` is choosing between options of a specific group. It also offers a lot of flexibility like selecting multiple options
3. `Select` should really be used for basic form selections. It is meant to be used in a `FormGroup` more than anything.

Hope that helps.

**Reply by raquelbreternitz** [reply]
Thank you, Joe!
